src/views/base.html

```diff
@@ -1,53 +1,46 @@
-    <meta property="og:url" content="{{ requestedUrl.href }}" />
-    <meta property="og:description" content="Join us for 24 hours of building, breaking and creating in the heart of Cambridge from January 28 – 29." />
-    <meta property="og:image" content="{{ requestedUrl.protocol }}//{{ requestedUrl.host }}{{ asset('images/fb-share.jpg') }}" />
-
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-    <meta name="language" content="en">
-    <meta name="theme-color" content="{{ colors.PRIMARY_COLOR }}">
-    <script src="https://use.typekit.net/jig8uxt.js"></script>
-    <script>try{Typekit.load({ async: true });}catch(e){}</script>
-    {% if browserSync %}{{ browserSync | safe }}{% endif %}
-    <link rel="apple-touch-icon" href="{{ asset('images/apple-touch-icon.png') }}" />
-    <link rel="stylesheet" type="text/css" href="{{ asset('styles/main.css') }}" />
-  </head>
-  <body>
-    {% block content %}
-    {% endblock %}
-
-    {% block scripts %}
-    <script type="text/javascript" src="{{ asset('scripts/main.js') }}"></script>
-
-    <!-- Facebook Pixel Code -->
-    <script>
-    !function(f,b,e,v,n,t,s){if(f.fbq)return;n=f.fbq=function(){n.callMethod?
-    n.callMethod.apply(n,arguments):n.queue.push(arguments)};if(!f._fbq)f._fbq=n;
-    n.push=n;n.loaded=!0;n.version='2.0';n.queue=[];t=b.createElement(e);t.async=!0;
-    t.src=v;s=b.getElementsByTagName(e)[0];s.parentNode.insertBefore(t,s)}(window,
-    document,'script','//connect.facebook.net/en_US/fbevents.js');
-    fbq('init', '412352858960781');
-    fbq('track', "PageView");</script>
-    <noscript><img height="1" width="1" style="display:none"
-    src="https://www.facebook.com/tr?id=412352858960781&ev=PageView&noscript=1"
-    /></noscript>
-
-    <!-- Google Analytics -->
-    <script>
-      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-      ga('create', 'UA-2529322-33', 'auto');
-      ga('send', 'pageview');
-    </script>
-    {% endblock %}
-  </body>
+	<head>
+		<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+		<title>{% block pageTitle %}{{ title }}{% endblock %}</title>
+		<meta name="title" content="{{ title }}" />
+		<meta name="description" content="{{ description }}" />
+		<meta name=keywords content="{{ keywords }}">
+		<meta name=author content="Hack Cambridge Team">
+		<meta name="language" content="en">
+		<meta name="viewport" content="width=device-width, user-scalable=no" />
+		<meta property="og:title" content="{{ title }}" />
+		<meta property="og:url" content="{{ requestedUrl.href }}" />
+		<meta property="og:description" content="{{ description }}" />
```

Comment:
It might be useful to have a default value for keywords and description so we can override them if we wish but don't have to specify them on every page.  We could do this by using the [default function](https://mozilla.github.io/nunjucks/templating.html#default-value-default-boolean) in Nunjucks.